components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/dao/impl/ApiFileDAOImpl.java

```diff
@@ -0,0 +1,486 @@
+/*
```

Comment:
What is the use case trying to achieve here? If we add this we have to maintain same code in two different places.